shared/rakefiles/xk_util.rake

```diff
@@ -220,4 +220,27 @@ task :revoke_org_admin => [@gcp_creds_file, :configure_extra_tf_vars] do
   end
 end
 
+# This task restores a list of images in snapshots. All the snapshots are stored
+# in the actual region of the project in the zone A.
```

Comment:
This is dangerous - some regions do not have zone a (like us-east1 or europe-west1).

Easy fix - detect first available zone for given region, more clever way - detect what zones are used by cluster and spread the snapshots across those zones.

Reply:
I wasn't aware of that fact. I'll fix this.